### PR TITLE
Improvement when using DISABLE_ANIMATIONS=true

### PR DESCRIPTION
**Pull request by cyril-sf**
@maddiehorowitz @Dylnuge @thangdinh @azirbel

### CODE REVIEW

File: tests/unit/modal_test.coffee [hunk 1]
```diff
@@ -0,0 +1,19 @@
+moduleForComponent 'modal', '[Unit] Modal component',
+  needs: [
+    'template:modal'
+    'template:modal_header'
+    'template:modal-footer'
+  ],
+  
+  teardown: ->
+    Ember.Widgets.DISABLE_ANIMATIONS = no
```
Comment (azirbel) [inline tests/unit/modal_test.coffee]: `modal.destroy()` ?

File: tests/unit/modal_test.coffee [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+test 'With DISABLE_ANIMATIONS=true, hide can be called several times', ->
```
Comment (azirbel) [inline tests/unit/modal_test.coffee]: `expect 1`?

File: tests/unit/modal_test.coffee [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  Ember.Widgets.DISABLE_ANIMATIONS = yes
+  modal = @subject()
+
+  @append()
+  andThen ->
+    modal.hide()
+    modal.hide()
+    ok yes, 'It works'
```
Comment (azirbel) [inline tests/unit/modal_test.coffee]: Maybe more verbose, repeating "hide can be called several times"?

**Comment by azirbel** [comment]
LGTMBC

File: tests/unit/modal_test.coffee (hunk 1 above)
Comment (cyril-sf) [inline tests/unit/modal_test.coffee]: It's already destroyed in the test, but that's not a good reason.I would hope that Ember QUnit does it for us. 

I'll check that.

**Pull request merged**